File: sort-python/CR.py
class cr:
    def crsort(self,a):
        for i in range(1,len(a)):
            for j in range(0,i):
                if a[i]<a[j]:
                    tmp=a[i]
                    for k in range(i,j,-1):
                        a[k]=a[k-1]
                    a[j]=tmp
        return a

File: sort-java/src/CR.java
public class CR {
    public CR(){};
    public int[] sort(int[] a){
        for (int i=1;i<a.length;i++){
            for (int j=0;j<i;j++){
                if (a[i]<a[j]){
                    int tmp=a[i];
                    for (int k=i;k>j;k--){
                        a[k]=a[k-1];
                    }
                    a[j]=tmp;
                }
            }
        }
        return a;
    }
}
